packages/dev-utils/lib/manifest-parser/impl.ts

```diff
@@ -14,14 +14,18 @@ function convertToFirefoxCompatibleManifest(manifest: Manifest) {
     ...manifest,
   } as { [key: string]: unknown };
 
-  manifestCopy.background = {
-    scripts: [manifest.background?.service_worker],
-    type: 'module',
-  };
-  manifestCopy.options_ui = {
-    page: manifest.options_page,
-    browser_style: false,
-  };
+  if (manifest.background?.service_worker) {
+    manifestCopy.background = {
+      scripts: [manifest.background?.service_worker],
```

Comment:
```suggestion
      scripts: [manifest.background.service_worker],
```

It doesn't need `?` because you already know this exists